api/Document.json

```diff
@@ -1658,10 +1658,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/copy_event",
           "support": {
             "chrome": {
-              "version_added": "58"
+              "version_added": "1"
```

Comment:
Didn't know these events were this old, but I tested http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8654 back in Chrome 16 on macOS and the event does fire, so this seems right. The `ClipboardEvent` interface is much newer though, the event was originally a plain `Event`.